From d4ec8861c07b34d1d05efaa09963fd742c322411 Mon Sep 17 00:00:00 2001
From: Jan Holesovsky <kendy@suse.cz>
Date: Fri, 14 May 2010 17:09:26 +0200
Subject: [PATCH 759/768] toolbar-decorations-svx.diff

---
 svx/source/tbxctrls/extrusioncontrols.src |   28 ++++------------------------
 svx/source/tbxctrls/fontworkgallery.src   |   14 ++------------
 svx/source/tbxctrls/linectrl.cxx          |    4 ++--
 svx/source/tbxctrls/tbcontrl.cxx          |    6 +++---
 4 files changed, 11 insertions(+), 41 deletions(-)

#diff --git svx/source/tbxctrls/extrusioncontrols.src svx/source/tbxctrls/extrusioncontrols.src
#index 37447b3..ba4f92d 100644
#--- svx/source/tbxctrls/extrusioncontrols.src
#+++ svx/source/tbxctrls/extrusioncontrols.src
#@@ -32,15 +32,10 @@
# 
# #define MASKCOLOR	MaskColor = Color { Red=0xFFFF; Green=0x0000; Blue=0xFFFF; };
# 
#-DockingWindow RID_SVXFLOAT_EXTRUSION_DIRECTION
#+FloatingWindow RID_SVXFLOAT_EXTRUSION_DIRECTION
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Extrusion Direction" ;
# 
#@@ -202,15 +197,10 @@ FloatingWindow RID_SVXFLOAT_EXTRUSION_DIRECTION
#     };
# };
# 
#-DockingWindow RID_SVXFLOAT_EXTRUSION_DEPTH
#+FloatingWindow RID_SVXFLOAT_EXTRUSION_DEPTH
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Extrusion Depth" ;
#     
#@@ -291,15 +281,10 @@ FloatingWindow RID_SVXFLOAT_EXTRUSION_DEPTH
# };
# 
# 
#-DockingWindow RID_SVXFLOAT_EXTRUSION_LIGHTING
#+FloatingWindow RID_SVXFLOAT_EXTRUSION_LIGHTING
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Extrusion Lighting" ;
#     
#@@ -607,15 +592,10 @@ FloatingWindow RID_SVXFLOAT_EXTRUSION_LIGHTING
#     };
# };
# 
#-DockingWindow RID_SVXFLOAT_EXTRUSION_SURFACE
#+FloatingWindow RID_SVXFLOAT_EXTRUSION_SURFACE
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Extrusion Surface" ;
#     
#diff --git svx/source/tbxctrls/fontworkgallery.src svx/source/tbxctrls/fontworkgallery.src
#index 7d5b20e..8bc9c94 100644
#--- svx/source/tbxctrls/fontworkgallery.src
#+++ svx/source/tbxctrls/fontworkgallery.src
#@@ -94,15 +94,10 @@ ModalDialog RID_SVX_MDLG_FONTWORK_GALLERY
#     };
# };
# 
#-DockingWindow RID_SVXFLOAT_FONTWORK_ALIGNMENT
#+FloatingWindow RID_SVXFLOAT_FONTWORK_ALIGNMENT
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Fontwork Alignment" ;
#     
#@@ -229,15 +224,10 @@ FloatingWindow RID_SVXFLOAT_FONTWORK_ALIGNMENT
#     };
# };
# 
#-DockingWindow RID_SVXFLOAT_FONTWORK_CHARSPACING
#+FloatingWindow RID_SVXFLOAT_FONTWORK_CHARSPACING
# {
#-    Border = FALSE ;
#+    StdPopup = TRUE ;
#     Hide = TRUE ;
#-    SVLook = TRUE ;
#-    Sizeable = FALSE ;
#-    Moveable = TRUE ;
#-    Closeable = TRUE ;
#-    Zoomable = TRUE ;
# 
#     Text [ en-US ] = "Fontwork Character Spacing" ;
#     
diff --git svx/source/tbxctrls/linectrl.cxx svx/source/tbxctrls/linectrl.cxx
index f3a21c1..bd88600 100644
--- svx/source/tbxctrls/linectrl.cxx
+++ svx/source/tbxctrls/linectrl.cxx
@@ -382,7 +382,7 @@ SvxLineEndWindow::SvxLineEndWindow(
     const String& rWndTitle ) :
     SfxPopupWindow( nSlotId,
                     rFrame,
-                    WinBits( WB_BORDER | WB_STDFLOATWIN | WB_SIZEABLE | WB_3DLOOK ) ),
+                    WinBits( WB_STDPOPUP ) ),
     pLineEndList	( NULL ),
     aLineEndSet		( this, WinBits( WB_ITEMBORDER | WB_3DLOOK | WB_NO_DIRECTSELECT ) ),
     nCols			( 2 ),
@@ -404,7 +404,7 @@ SvxLineEndWindow::SvxLineEndWindow(
     SfxPopupWindow( nSlotId,
                     rFrame,
                     pParentWindow,
-                    WinBits( WB_BORDER | WB_STDFLOATWIN | WB_SIZEABLE | WB_3DLOOK ) ),
+                    WinBits( WB_STDPOPUP ) ),
     pLineEndList	( NULL ),
     aLineEndSet		( this, WinBits( WB_ITEMBORDER | WB_3DLOOK | WB_NO_DIRECTSELECT ) ),
     nCols			( 2 ),
diff --git svx/source/tbxctrls/tbcontrl.cxx svx/source/tbxctrls/tbcontrl.cxx
index db1c542..01d02a6 100644
--- svx/source/tbxctrls/tbcontrl.cxx
+++ svx/source/tbxctrls/tbcontrl.cxx
@@ -829,7 +829,7 @@ SvxColorWindow_Impl::SvxColorWindow_Impl( const OUString&            rCommand,
                                           const String&              rWndTitle,
                                           Window*                    pParentWindow ) :
 
-    SfxPopupWindow( nSlotId, rFrame, pParentWindow, WinBits( WB_BORDER | WB_STDFLOATWIN | WB_3DLOOK|WB_DIALOGCONTROL ) ),
+    SfxPopupWindow( nSlotId, rFrame, pParentWindow, WinBits( WB_STDPOPUP ) ),
 
     theSlotId( nSlotId ),
     aColorSet( this, WinBits( WB_ITEMBORDER | WB_NAMEFIELD | WB_3DLOOK | WB_NO_DIRECTSELECT) ),
@@ -1055,7 +1055,7 @@ void SvxColorWindow_Impl::StateChanged( USHORT nSID, SfxItemState eState, const
 
 SvxFrameWindow_Impl::SvxFrameWindow_Impl( USHORT nId, const Reference< XFrame >& rFrame, Window* pParentWindow ) :
 
-    SfxPopupWindow( nId, rFrame, pParentWindow, WinBits( WB_BORDER | WB_STDFLOATWIN | WB_3DLOOK | WB_DIALOGCONTROL ) ),
+    SfxPopupWindow( nId, rFrame, pParentWindow, WinBits( WB_STDPOPUP ) ),
     aFrameSet   ( this, WinBits( WB_ITEMBORDER | WB_DOUBLEBORDER | WB_3DLOOK | WB_NO_DIRECTSELECT ) ),
     bParagraphMode(sal_False)
 
@@ -1329,7 +1329,7 @@ BOOL SvxFrameWindow_Impl::Close()
 
 SvxLineWindow_Impl::SvxLineWindow_Impl( USHORT nId, const Reference< XFrame >& rFrame, Window* pParentWindow ) :
 
-    SfxPopupWindow( nId, rFrame, pParentWindow, WinBits( WB_BORDER | WB_STDFLOATWIN | WB_3DLOOK | WB_DIALOGCONTROL ) ),
+    SfxPopupWindow( nId, rFrame, pParentWindow, WinBits( WB_STDPOPUP ) ),
 
     aLineSet( this, WinBits( WB_3DLOOK | WB_ITEMBORDER | WB_DOUBLEBORDER | WB_NAMEFIELD | WB_NONEFIELD | WB_NO_DIRECTSELECT ) )
 {
-- 
1.7.0.1

